jaeger/charts/linkerd-jaeger/values.yaml

```diff
@@ -184,6 +183,18 @@ webhook:
     #matchLabels:
     #  foo: bar
 
+  resources:
+    cpu:
+      # -- Maximum amount of CPU units that the jaeger container can use
+      limit:
+      # -- Amount of CPU units that the jaeger container requests
+      request:
+    memory:
+      # -- Maximum amount of memory that jaeger container can use
+      limit:
+      # -- Amount of memory that the jaeger container requests
+      request:
```

Comment:
Can you please change the comments here, as they should refer to the webhook container, and not the jaeger container?